templates/details_data_product.html

```diff
@@ -91,21 +91,15 @@ <h2 class="govuk-heading-l">{{result.name}}</h2>
                 </tr>
               </thead>
               <tbody class="govuk-table__body">
-                <tr class="govuk-table__row">
-                  <th scope="row" class="govuk-table__header">Table 1</th>
-                  <td class="govuk-table__cell">This is description about the table</td>
-                  <td class="govuk-table__cell">  <a href="#" class="govuk-link">Schema details</a></td>
-                </tr>
-                <tr class="govuk-table__row">
-                  <th scope="row" class="govuk-table__header">Table 2</th>
-                  <td class="govuk-table__cell">This is description about the table</td>
-                  <td class="govuk-table__cell">  <a href="#" class="govuk-link">Schema details</a></td>
-                </tr>
-                <tr class="govuk-table__row">
-                  <th scope="row" class="govuk-table__header">Table 3</th>
-                  <td class="govuk-table__cell">This is description about the table</td>
-                  <td class="govuk-table__cell">  <a href="#" class="govuk-link">Schema details</a></td>
-                </tr>
+                {% for table in tables %}
+                {% with table_type=table.type|lower %}
+                  <tr class="govuk-table__row">
+                    <th scope="row" class="govuk-table__header">{{table.name}}</th>
+                    <td class="govuk-table__cell">{{table.description}}</td>
```

Comment:
I think this should be passed through the markdown filter so we can have rich text descriptions for tables as well as data products. What do you think?